packages/terra-functional-testing/lib/services/wdio-docker-service.js

```diff
@@ -0,0 +1,164 @@
+/* eslint-disable class-methods-use-this */
+const path = require('path');
+const util = require('util');
+const childProcess = require('child_process');
+const Logger = require('../logger/logger');
+
+const exec = util.promisify(childProcess.exec);
+const logger = new Logger({ prefix: 'wdio-docker-service' });
+
+class DockerService {
+  /**
+   * Prepares the docker testing environment.
+   */
+  async onPrepare(config) {
```

Comment:
Should we check if docker is installed? And fail if it isn't?, just thinking of questions we've had in the past.

Reply:
I'll look into adding this check. We might end up in a weird situation, but at least we'll be able to log something to the console. We can't fail the test run in the services, so the service will fail and then the tests will attempt to run. 

https://github.com/webdriverio/webdriverio/issues/5398

Reply:
Added a check that will fail if docker is not installed:

https://github.com/cerner/terra-toolkit/pull/412/commits/7d1d57f6fcc0a7f590c79a38a6222021d70786e6